src/services/pscExtensionService.ts

```diff
@@ -59,3 +59,73 @@ export const createPscExtension = async (request: Request, transactionId: string
 
     return castedSdkResponse;
 };
+
+export const getIsPscExtensionValid = async (request: Request, transactionId: string, pscNotificationId: string, companyNumber: string): Promise<ValidationStatusResponse> => {
+    if (!transactionId) {
+        throw new Error(`Aborting: transactionId is required for PSC Extension validation GET request`);
+    }
+    if (!pscNotificationId) {
+        throw new Error(`Aborting: pscNotificationId is required for PSC Extension validation GET request`);
+    }
+    if (!companyNumber) {
+        throw new Error(`Aborting: companyNumber is required for PSC Extension validation GET request`);
+    }
+
+    const oAuthApiClient: ApiClient = createOAuthApiClient(request.session);
+
+    logger.debug(`Getting PSC Extension validation for transactionId="${transactionId}", pscNotificationId="${pscNotificationId}", companyNumber="${companyNumber}"`);
```

Comment:
if you implement the refactor above you can just stringily the params i.e
```suggestion
    const params = { transactionId, pscNotificationId, companyNumber }; // from above (prior refactor)
    logger.debug(`Getting PSC Extension validation for ${JSON.stringify(params)}`);
```